src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java

```diff
@@ -101,4 +104,73 @@ public void selezioneVediDettaglioButton() {
     public String getTextLinkAttestazioniOpponibili(int i) {
         return attestazioniFile.get(i).getText();
     }
+
+    public void checkTestoBoxPagamento(By xpath, Boolean isTrue) {
+        try {
+            if (isTrue){
+                getWebDriverWait(10).withMessage("campo non trovato").until(ExpectedConditions.visibilityOfElementLocated(xpath));
+            }else {
+                getWebDriverWait(10).withMessage("campo  trovato").until(ExpectedConditions.invisibilityOfElementLocated(xpath));
+            }
+        } catch (TimeoutException e) {
+            logger.error("il box pagamento non é stato caricato correttamente: " + e.getMessage());
+            Assert.fail("il box pagamento non é stato caricato correttamente: " + e.getMessage());
+        }
+    }
+
+    public void waitLoadDettaglioNotificaAnnullataDESection() {
+        try {
+            boolean isSelfcare = driver.getCurrentUrl().contains("selfcare");
+            By titleDettaglioNotificaField = By.id("title-of-page");
+            By statoNotificaBy = By.id("notification-state");
+            By indietroButtonBy = By.id("breadcrumb-indietro-button");
+            By informazioniBy = By.id("notification-detail-table");
+            By allegatiSection = By.id("notification-detail-document-attached");
+            By sezioneRecapiti = By.id("side-item-I tuoi recapiti");
+            By altriDocumenti = isSelfcare ? By.xpath("//div[@data-testid='aarDownload']") : By.xpath("//div[@data-testid='aarBox']");
```

Comment:
You can simple adjust this by using selenium methods given.
```suggestion
            By aarDownload =  By.xpath("//div[@data-testid='aarDownload']");
            By aarBox = By.xpath("//div[@data-testid='aarBox']");
            
            getWebDriverWait(10).withMessage("La sezione recapiti non è visibile").until(ExpectedConditions.or(
            ExpectedConditions.visibilityOfElementLocated(aarDownload),
            ExpectedConditions.visibilityOfElementLocated(aarBox)))
```

Reply:
done

Reply:
change hasn't been done